src/symbolize/libbacktrace.rs

```diff
@@ -347,6 +347,28 @@ unsafe fn init_state() -> *mut bt::backtrace_state {
                 load_filename_opt().unwrap_or(ptr::null())
             }
 
+            #[cfg(target_vendor = "uwp")]
+            unsafe fn create_file(name: &[i8]) -> HANDLE {
+                CreateFile2(name.as_ptr(),
+                            GENERIC_READ,
+                            FILE_SHARE_READ | FILE_SHARE_WRITE,
+                            OPEN_EXISTING,
+                            ptr::null_mut()
+                )
+            }
+
+            #[cfg(target_vendor = "pc")]
```

Comment:
FWIW this I think would probably be best as `#[cfg(not(...))]` instead of `target_vendor = "pc"`